types/bookshelf/index.d.ts

```diff
@@ -373,6 +373,7 @@ declare namespace Bookshelf {
     interface SerializeOptions {
         shallow?: boolean | undefined;
         omitPivot?: boolean | undefined;
+        hidden?: array | undefined;
```

Comment:
Shouldn't it be:

```suggestion
        hidden?: string[];
```

?